### DISCUSSION TITLE
What's the latest stepper component?

### DISCUSSION THREAD

**Discussion by pamelahu** [twilio-labs/paste#3194] [Q&A] [answered]
Hi team! I can't seem to find this component in the Paste website...
<img width="1057" alt="Screenshot 2023-04-25 at 12 26 18 PM" src="https://user-images.githubusercontent.com/18605252/234341662-4cc54545-3583-44f5-a111-5b87c6621580.png">

Curious if it's just a naming thing I'm not knowing but also I remember seeing a couple variations floating around. I'm wondering what the latest version of this component is (using it for onboarding in Athena.)

**Comment by loreina** [comment, marked as answer]
Hi @pamelahu,

Paste doesn't currently have a Stepper component but it's well requested by product teams and in our backlog. The best way to help us prioritize the component is to [file a dependency request](https://airtable.com/shrvMvL3BG6Zm0f9l). That way our team is aware of your timeline and can follow up on our progress. In the meantime, your team can implement your own stepper and when Paste releases a Stepper, swap to the Paste component then. We plan to work on the Stepper in Q3.

It might also be nice to sync with @dreamealist who worked on a Stepper that's in production https://github.com/twilio-labs/paste/discussions/3087 and see if there's opportunity to reuse the engineering implementation.
